Makefile

```diff
@@ -5,29 +5,31 @@ data/external/deep/glove.840B.300d.txt:
 	rm /tmp/glove.840B.300d.zip
 
 output/deep/params: data/external/deep/glove.840B.300d.txt
-	python3 guesser/util/format_dan.py --threshold=5
-	python3 guesser/util/load_embeddings.py
-	python3 guesser/dan.py
+	mkdir -p data/external/deep/train
+	python3 qanta/guesser/util/format_dan.py --threshold=5
+	python3 qanta/guesser/util/load_embeddings.py
+	python3 qanta/guesser/dan.py
 
 output/kenlm.binary:
 	mkdir -p temp
-	python3 cli.py build_mentions_lm_data data/external/wikipedia /tmp/wiki_sent
+	python3  cli.py build_mentions_lm_data data/external/wikipedia /tmp/wiki_sent
 	lmplz -o 5 < /tmp/wiki_sent > output/kenlm.arpa
 	build_binary output/kenlm.arpa $@
 	rm /tmp/wiki_sent
 
-output/wikifier/data/input/:
+output/wikifier/data/input:
```

Comment:
I don't quite understand the difference here for makefiles, any reference to see the semantics?